.docs/user-guide/docker-plugins.md

```diff
@@ -80,18 +80,118 @@ Environment Variable | Description | Default | Required
 `EBS_REGION` | The AWS region | `us-east-1` |
 
 ### Elastic File System
+The EFS plug-in can be installed with the following command:
+
+```bash
+$ docker plugin install rexray/efs \
+  EFS_ACCESSKEY=abc \
+  EFS_SECRETKEY=123 \
+  EFS_SECURITYGROUPS="sg-123 sg-456" \
+  EFS_TAG=rexray
+```
+
+#### Requirements
+The EFS plug-in requires that nfs utilities be installed on the
+same host on which Docker is running. You should be able to mount an
+nfs export to the host.
+
+#### Privileges
+The EFS plug-in requires the following privileges:
+
+Type | Value
+-----|------
+network | `host`
+mount | `/dev`
+allow-all-devices | `true`
+capabilities | `CAP_SYS_ADMIN`
+
+#### Configuration
+The following environment variables can be used to configure the EFS
+plug-in:
+
+Environment Variable | Description | Default | Required
+---------------------|-------------|---------|---------
+`EFS_ACCESSKEY` | The AWS access key | | ✓
+`EFS_SECRETKEY` | The AWS secret key | | ✓
+`EFS_REGION` | The AWS region | |
+`EFS_SECURITYGROUPS` | The AWS security groups to bind to | `default` |
+`EFS_TAG` | Only consume volumes with tag (tag\volume_name)| |
+`EFS_DISABLESESSIONCACHE` | new AWS connection is established with every API call | `false` |
 
-### Simple Storage Service
+### Simple Storage Service (S3)
```

Comment:
Is any of our documention linking here directly? if so, the link to it will break (unless you add of the new hidden anchors).  As this stuff is so new, I don't think there are any links to this stuff yet.